package.json

```diff
@@ -15,6 +15,7 @@
     "@ethersproject/providers": "^5.6.4",
     "@ethersproject/wallet": "^5.6.2",
     "cross-fetch": "^3.1.5",
+    "ethers": "^5.6.9",
```

Comment:
It's better to only import the sub package that we use here

Reply:
fixed